dist.ini

```diff
@@ -0,0 +1,69 @@
+name    = Time-Duration-Concise-Localize
+author  = binary.com <BINARY@cpan.org>
+license = Perl_5
+copyright_holder = binary.com
+copyright_year   = 2017
+main_module = lib/Time/Duration/Concise/Localize.pm
+[Git::GatherDir]
+exclude_filename = Makefile.PL
+include_dotfiles = 1
+[PruneCruft]
+except = t/rc/\.perl*rc$
+[ManifestSkip]
+[MetaYAML]
+[License]
+[Readme]
+[MakeMaker]
+eumm_version = 6.48
+prereq_fatal = 1
+[ExecDir]
+[ShareDir]
+dir = share
+[Manifest]
+[TestRelease]
+[ConfirmRelease]
+[Prereqs::FromCPANfile]
+[Prereqs / BuildRequires]
+perl = 5.014000
+[CheckPrereqsIndexed]
+[CheckExtraTests]
+[NextRelease]
+[VersionFromModule]
+[ReversionOnRelease]
+[OurPkgVersion]
+[GitHub::Meta]
+repo = binary-com/perl-Time-Duration-Concise-Localize
+[InstallGuide]
+[MetaJSON]
+[InsertExample]
+[PerlTidy]
+perltidyrc = t/rc/.perltidyrc
+[PodSyntaxTests]
+[MojibakeTests]
+[Test::CheckDeps]
+[Test::Compile]
+[Test::Synopsis]
+[Test::EOL]
+[Test::Version]
+[Test::Pod::LinkCheck]
+[PodCoverageTests]
+;[Test::UnusedVars]
```

Comment:
There is a bug in this plugin for module 'Concise.pm'. So I disabled it. Please see the comment in `Concise.pm`